SPECS/autoconf213/autoconf213.spec

```diff
@@ -0,0 +1,268 @@
+Summary:    A GNU tool for automatically configuring source code
+Name:       autoconf213
+Version:    2.13
+Release:    49%{?dist}
+License:    GPLv2+
+URL:        http://www.gnu.org/software/autoconf/
+Source:     https://ftp.gnu.org/gnu/autoconf/autoconf-%{version}.tar.gz
+Patch0:     autoconf-2.13-versioning.patch
+Patch1:     autoconf213-destdir.patch
+Patch2:     autoconf213-info.patch
+Vendor:     Microsoft Corporation
+Distribution:  Azure Linux
+
+Requires:   gawk, m4 >= 1.1, coreutils
+BuildRequires:   texinfo, m4 >= 1.1, perl, gawk, dejagnu, flex
+BuildArch:  noarch
+
+%description
+GNU's Autoconf is a tool for configuring source code and Makefiles.
+Using Autoconf, programmers can create portable and configurable
+packages, since the person building the package is allowed to specify
+various configuration options.
+
+You should install Autoconf if you are developing software and you
+would like to use it to create shell scripts that will configure your
+source code packages. If you are installing Autoconf, you will also
+need to install the GNU m4 package.
+
+Note that the Autoconf package is not required for the end-user who
+may be configuring software with an Autoconf-generated script;
+Autoconf is only required for the generation of the scripts, not their
+use.
+
+%prep
+%setup -q -n autoconf-%{version}
+%patch -P 0 -p1
+%patch -P 1 -p1
+%patch -P 2 -p1
+
+mv autoconf.texi autoconf213.texi
+rm -f autoconf.info
+
+%build
+%configure --program-suffix=-%{version}
+make
+
+%install
+rm -rf ${RPM_BUILD_ROOT}
+make install DESTDIR=$RPM_BUILD_ROOT
+
+rm ${RPM_BUILD_ROOT}/%{_bindir}/autoscan-%{version}
+
+# We don't want to include the standards.info stuff in the package,
+# because it comes from binutils...
+rm -f ${RPM_BUILD_ROOT}%{_infodir}/standards*
+
+%check
+# Skip AC_FUNC_GETLOADAVG which fails to find /dev/kmem due to CONFIG_DEVKMEM not being set
+sed -i 's/AC_DEFUN(AC_FUNC_GETLOADAVG,/AU_DEFUN(AC_FUNC_GETLOADAVG,/g' ./acspecific.m4
+# Skip Fortan 77 (F77) tests which are not supported with our version of gcc
+sed -i 's/AC_DEFUN(AC_PROG_F77,/AU_DEFUN(AC_PROG_F77,/g' ./acspecific.m4
+sed -i 's/AC_DEFUN(AC_PROG_F77_WORKS,/AU_DEFUN(AC_PROG_F77_WORKS,/g' ./acspecific.m4
+sed -i 's/AC_DEFUN(AC_F77_LIBRARY_LDFLAGS,/AU_DEFUN(AC_F77_LIBRARY_LDFLAGS,/g' ./acspecific.m4
+make check
+
+%post
+/sbin/install-info %{_infodir}/%{name}.info %{_infodir}/dir || :
+
+%preun
+if [ "$1" = 0 ]; then
+    /sbin/install-info --delete %{_infodir}/%{name}.info %{_infodir}/dir || :
+fi
+
+%files
+%defattr(-,root,root,-)
+%license COPYING
+%{_bindir}/*
+%{_infodir}/*.info*
+%{_datadir}/autoconf-%{version}/
+
+%changelog
+* Fri Jul 1 2022 Jamie Magee <jamagee@microsoft.com> - 2.13-49
```

Comment:
Did we miss a changelog entry or this is simple `lift & shift` restore ?

Reply:
I didn't add a new changelog entry - just restored as it was